applications/FemToDemApplication/FemToDemApplication.py

```diff
@@ -1,10 +1,3 @@
-# makes KratosMultiphysics backward compatible with python 2.6 and 2.7
-from __future__ import print_function, absolute_import, division
-
-from KratosMultiphysics.MeshingApplication import *
-from KratosMultiphysics.PfemFluidDynamicsApplication import *
-from KratosMultiphysics.DEMApplication import *
-from KratosMultiphysics.StructuralMechanicsApplication import *

```

Comment:
I looked into your CMakeLists and saw that you link against those apps:
https://github.com/KratosMultiphysics/Kratos/blob/f7909a7e675003393e6a55ae76bd085758e61205/applications/FemToDemApplication/CMakeLists.txt#L73

I.e. if you do not import them here (before importing `KratosFemToDemApplication`) it will crash unless you imported those apps somewhere before
I.e. for link-time dependencies it is strongly recommended importing the apps you depend on before you import the actual app core

Reply:
Can I import them here?

Reply:
Sure!
Check e.g. what the RANS-app does (which links to FluidDynamics)
https://github.com/KratosMultiphysics/Kratos/blob/f7909a7e675003393e6a55ae76bd085758e61205/applications/RANSApplication/RANSApplication.py#L9-L10

Reply:
I'll do it that way THX